life/modernized/04-chapter-3.adoc

```diff
@@ -67,6 +67,8 @@ they were asked their names and country.
 We will give the rest in Francis Howgill`'s words,
 contained in a letter to Margaret Fell, which was signed by Edward Burrough.
 
+[quote]
```

Comment:
This is from a letter to Margaret Fell, so it should probably be [.embedded-content-document.letter] instead of a quote. 👍🏻